src/pybricks/hubs.pyi

```diff
@@ -29,7 +29,7 @@ class TechnicHub:
 
 
 class PrimeHub:
-    def __init__(self, top_size: Axis, front_side: Axis): ...
+    def __init__(self, top_size: Axis = Axis.Z, front_side: Axis = Axis.X): ...
```

Comment:
While you're updating this anyway: `top_size` should be `top_side`.

Reply:
Now, that's the typo! And also the reason why I had a few typos like that in my code. Now I realise that it was the IDE auto-suggesting that for me 😊

Reply:
Fixed.

Reply:
Thanks!